openpype/hosts/nuke/startup/menu.py

```diff
@@ -1,15 +1,42 @@
+import os
 from openpype.hosts.nuke.api.lib import (
     on_script_load,
     check_inventory_versions,
     WorkfileSettings
 )
+from openpype.lib.path_tools import HostDirmap
+from openpype.settings import get_project_settings
+from openpype.modules import ModulesManager
 
 import nuke
 from openpype.api import Logger
 
 log = Logger().get_logger(__name__)
 
 
+class NukeDirmap(HostDirmap):
```

Comment:
can this and most of class and def definitions be moved into `/nuke/api/lib.py` ? we can keep only calback activators here.